df-server/src/main/java/com/hashmapinc/tempus/witsml/server/api/StoreImpl.java

```diff
@@ -219,26 +219,31 @@ public WMLS_GetFromStoreResponse getFromStore(
         try {
```

Comment:
2 basic changes here:
- added support for the ValveUser for passing credentials to the query context
- call `valve.getObject(qc)` to get the XML string to return in the response object formatted according to the `CLIENT_VERSION` in `qc`